mappings/net/minecraft/client/gui/AbstractCommandBlockScreen.mapping

```diff
@@ -1,21 +1,61 @@
 CLASS dak net/minecraft/client/gui/AbstractCommandBlockScreen
+	CLASS dak$a SuggestionWindow
+		FIELD b area Ldkt;
+		FIELD c suggestions Lcom/mojang/brigadier/suggestion/Suggestions;
+		FIELD d typedText Ljava/lang/String;
+		FIELD e inWindowIndex I
+		FIELD f selection I
+		FIELD g mouse Lcru;
+		FIELD h chosen Z
```

Comment:
What does this mean?

Reply:
Means if a suggestion has been chosen, as if you have typed a tab after you have input the initial characters. When this is true, typing tab will go to next suggestion instead of filling this suggestion.

Reply:
Will do `completed`